app/src/features/apiClient/screens/apiClient/components/views/components/Collection/components/CollectionRunnerView/components/RunConfigView/RunConfigSettings/DataFileSelector.tsx

```diff
@@ -95,7 +99,14 @@ export const DataFileSelector: React.FC = () => {
     <>
       {!dataFile ? (
         <>
-          <RQButton size="small" icon={<MdOutlineFileUpload />} onClick={handleFileSelection}>
+          <RQButton
+            size="small"
+            icon={<MdOutlineFileUpload />}
+            onClick={() => {
+              trackCollectionRunnerSelectFileClicked();
```

Comment:
Won't it be better if handleFileSelection calls the tracker? Only one source for tracking call. WDYT?